src/main/java/com/horus/yoga/controller/UserController.java

```diff
@@ -26,14 +26,14 @@ public APIResponse<UserDTO> getById(@PathVariable UUID id) {
         return APIResponse.ok(user);
     }
 
-    @GetMapping("/")
+    @GetMapping
```

Comment:
why remove the slash? the slash is not unnecessary, it is used to make sure that you are on the next item

Reply:
There are no next item that's why I removed them from those only.
If we check any website online, we can notice that no extra slash is used.